### ISSUE TITLE
Expecting value: line 1 column 1 (char 0) occurs by web api

### ISSUE THREAD

**Issue by zahra-ash0uri** [ping/instagram_private_api#245]
As I use web client without login authentication, I faced to this exception while using endpoints such as media_comments and user_info:
```python
Expecting value: line 1 column 1 (char 0) 
```
First I guessed it may caused by python request library, or something about network issues so I tried to be more polite and slow in sending requests, but it did not work.
Any idea to get rid of that or where it originates?

**Comment by wlerin** [comment]
500/502 error? I get those sometimes when getting checking story reels, but they seem to be temporary. Although I'm use the private api so maybe your issue is unrelated.

**Comment by zahra-ash0uri** [comment]
> 500/502 error? I get those sometimes when getting checking story reels, but they seem to be temporary. Although I'm use the private api so maybe your issue is unrelated.

Yes, it's almost temporary but not about user_info! I get this error for every user_info request.

PS: As i don't use login credential and the proxy is getting user_info responses elsewhere, it may have concurrent requests while using this endpoint.

**Comment by thanhnguyen1121** [comment]
@zahra-ash0uri  I have same problems. Do you have a solution?

**Comment by hyper750** [comment]
Similar problem as https://github.com/ping/instagram_private_api/issues/266, frontend is using `/<username>/?__a=1` with login required instead of `/graphql/query/`. Any solution?

**Comment by zahra-ash0uri** [comment]
> Similar problem as #266, frontend is using `/<username>/?__a=1` with login required instead of `/graphql/query/`. Any solution?

What I understood up to now is that our problem solves with Graph api, I mean /graphql/query/?{query_hash}blah-blah. I couldn't find the specific query_hash which is used by endpoint user_info.
PS: Apparently every endpoint has it's own query_hash.

**Comment by EsmaeiliSina** [comment]
> 
> 
> > Similar problem as #266, frontend is using `/<username>/?__a=1` with login required instead of `/graphql/query/`. Any solution?
> 
> What I understood up to now is that our problem solves with Graph api, I mean /graphql/query/?{query_hash}blah-blah. I couldn't find the specific query_hash which is used by endpoint user_info.
> PS: Apparently every endpoint has its own query_hash.

This problem still exists
Do you have a solution to solve this @zahra-ash0uri?
